requirements.txt

```diff
@@ -1,4 +1,3 @@
 enum34>=1.1.6;python_version<"3.4"
```

Comment:
thinking we should just remove `enum` here too. I don't think we want to be supporting older versions of 3 either. Feel like at the minimum 3.5+